### PR TITLE
refactor(wallet-mobile): portfolio tabs

### PR DESCRIPTION
**Pull request by jorbuedo**
##### Ticket

[YOMO-1931](https://emurgo.atlassian.net/browse/YOMO-1931)

[YOMO-1931]: https://emurgo.atlassian.net/browse/YOMO-1931?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/features/Portfolio/common/PortfolioProvider.tsx [hunk 1]
```diff
@@ -2,16 +2,47 @@ import {invalid} from '@yoroi/common'
 import {produce} from 'immer'
 import * as React from 'react'
 
+export const PortfolioDetailsTab = {
+  Performance: 'Performance',
+  Overview: 'Overview',
+  Transactions: 'Transactions',
+} as const
+export type PortfolioDetailsTab = (typeof PortfolioDetailsTab)[keyof typeof PortfolioDetailsTab]
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/Portfolio/common/PortfolioProvider.tsx:10]: Isn't enum better in this case?

File: apps/wallet-mobile/src/features/Portfolio/common/PortfolioProvider.tsx [hunk 2]
```diff
@@ -45,17 +86,52 @@ export const PortfolioProvider = ({
...
 
-type PortfolioContextAction = {
-  type: PortfolioActionType.SetIsPrimaryTokenActive
+type SetIsPrimaryTokenActiveAction = {
+  type: typeof PortfolioActionType.SetIsPrimaryTokenActive
   payload: {isActive: boolean}
 }
 
+type SetDetailsTabAction = {
+  type: typeof PortfolioActionType.SetDetailsTab
```
Comment (michalszorad) [inline apps/wallet-mobile/src/features/Portfolio/common/PortfolioProvider.tsx:111]: is `typeof` needed here?

File: apps/wallet-mobile/src/features/Portfolio/common/PortfolioProvider.tsx (hunk 1 above)
Comment (jorbuedo) [reply apps/wallet-mobile/src/features/Portfolio/common/PortfolioProvider.tsx:10]: It's functionally the same, except `enum` isn't valid js and it gets transpired as madness, so it really should never be used 🤷🏻‍♂️

File: apps/wallet-mobile/src/features/Portfolio/common/PortfolioProvider.tsx (hunk 2 above)
Comment (jorbuedo) [reply apps/wallet-mobile/src/features/Portfolio/common/PortfolioProvider.tsx:111]: The way it is, yes

**Review by stackchain — APPROVED** [review]

**Pull request merged**